karma.conf.js

```diff
@@ -30,3 +39,15 @@ module.exports = function (config) {
     sourcemap: true
   });
 };
+
+// Helpers
+function JasmineSeedReporter(baseReporterDecorator) {
+  baseReporterDecorator(this);
+
+  this.onBrowserComplete = (browser, result) => {
+    const seed = result.order && result.order.random && result.order.seed;
+    if (seed) this.write(`${browser}: Randomized with seed ${seed}.\n`);
+  };
+
+  this.onRunComplete = () => undefined;
```

Comment:
Out of curiosity, what does this do?

I commented it out and the Total was printed out twice in the log.   Is that all, or is there something more labour-intensive happening in the background?
![image](https://user-images.githubusercontent.com/129736527/236412625-a71e3fd7-d7ad-48e3-948e-9d35227bb32c.png)

Reply:
Honestly not sure. I stripped the seed reporter stuff straight from the Angular docs site [here](https://github.com/angular/angular/blob/80fe08be96633171567e30eb91dd1d23c60a2952/aio/karma.conf.js)

Reply:
fair 'nuff